src/foam/dao/RoutingJDAO.js

```diff
@@ -32,6 +32,7 @@ foam.CLASS({
       of: 'foam.dao.RoutingJournal',
       name: 'journal',
       javaPostSet: `
+getJournal().setDAOForName(getService(), getDelegate());
```

Comment:
Is this the right place for this?  I think the EasyDAO and/or NSpec.serviceScript should do it.   The EasyDAO knows exactly which decorators should be before/after the JDAO.